### PR TITLE
Add the image icons for three types of the TwitterMap

### PR DESCRIPTION
**Pull request by ChankiWu**
Finished my part of frontend's work so qiushi and teyu can continue to add pointmap and heatmap !

Old UI:

image

New UI:

image

### CODE REVIEW

File: examples/twittermap/web/public/javascripts/sidebar/controllers.js [hunk 1]
```diff
@@ -53,4 +53,46 @@ angular.module('cloudberry.sidebar', ['cloudberry.common'])
...
     };
-  });
\ No newline at end of file
+  })
+
+    .controller('choosemap', function ($scope, $window, cloudberry, $rootScope) {
+
+        $scope.result = null;
+
+        //chanki
```
Comment (waans11) [inline examples/twittermap/web/public/javascripts/sidebar/controllers.js]: Please do not put a developer name here (unnecessary comment).

File: examples/twittermap/web/public/javascripts/sidebar/controllers.js [hunk 2]
```diff
@@ -53,4 +53,46 @@ angular.module('cloudberry.sidebar', ['cloudberry.common'])
...
+        icon1.addEventListener("click", function () {
+
+            $rootScope.maptype = 'countmap';
+            $rootScope.$emit("maptypeChange", $scope.maptype);
+
+        });
+
+        icon2.addEventListener("click", function () {
+
+            //Interface with teyuc
```
Comment (waans11) [inline examples/twittermap/web/public/javascripts/sidebar/controllers.js]: Please do not put a developer name here (unnecessary comment).

**Comment by waans11** [comment]
LGTM. You can merge this.

**Comment by chenlica** [comment]
@ChankiWu : Congrats for merging your first PR!

**Pull request merged**